cmd/skupper/skupper_unexpose_test.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/spf13/cobra"
+	"gotest.tools/assert"
+)
+
+func Test_cmdUnexpose(t *testing.T) {
```

Comment:
Sure! good point. Will remove the underscore.
Just to add context: (without any foundation other than my personal opinion) I started using the underscore for function or method names starting with "lowercase", because, you may have a method with the same name, but starting with upercase, and in that case... you have a problem with the testname... TestOther -> is the test for func other() or for func Other()...
Anyway this is not even the case.